src/Tasks/Common/Resources/Strings.resx

```diff
@@ -880,7 +880,7 @@ You may need to build the project on another operating system or architecture, o
     <comment>{StrBegin="NETSDK1193: "}</comment>
   </data>
   <data name="CannotHaveSolutionLevelOutputPath" xml:space="preserve">
-    <value>NETSDK1194: The "--output" option isn't supported when building a solution.</value>
+    <value>NETSDK1194: The "--output" option isn't supported when building a solution. Specifying a solution-level output path results in all projects copying outputs to the same directory, which can lead to inconsistent builds.</value>
```

Comment:
I'm assuming it's ok to ship a servicing fix with error message clarifications without accompanying localization? Do we just merge those when they come in later?

Reply:
Yes, I believe so.